### DISCUSSION TITLE
Way to tab without taking suggestion

### DISCUSSION THREAD

**Discussion by twigs67** [community/community#18878] [Copilot Conversations]
There are cases where Copilot suggest something, but I don't want that suggestion and just want to tab. Pressing tab takes the suggestion though. There has to be a way around this?

I'm using JetBrains on mac. 

Any help would be appreciated

**Comment by TheShark27** [comment]
You can press ESC to delete the current suggestion and tab freely. You'll need to do each time you want to tab.
